### PR TITLE
fix(TBD-11897): merge maintenance into master

### PR DESCRIPTION
**Pull request by talend-jphautin**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

TBD-11897

linked to PR https://github.com/Talend/tbd-studio-ee/pull/2282

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Comment by talend-jphautin** [comment]
build-patch

**Comment by talend-jphautin** [comment]
build-patch

**Review by lbourgeois — APPROVED** [review]
LGTM

File: main/components_lib/talend-lineage-api/.classpath [hunk 1]
```diff
@@ -10,11 +10,13 @@
 		<attributes>
 			<attribute name="optional" value="true"/>
 			<attribute name="maven.pomderived" value="true"/>
+			<attribute name="test" value="true"/>
```
Comment (AlixMetivier) [inline main/components_lib/talend-lineage-api/.classpath:13]: idk where that comes from buuuuuut :D

**Review by AlixMetivier — DISMISSED** [review]

File: main/components_lib/talend-lineage-api/.classpath (hunk 1 above)
Comment (talend-jphautin) [reply main/components_lib/talend-lineage-api/.classpath:13]: removed

**Pull request merged**